doc/devices/numpy_wavefunction.rst

```diff
@@ -0,0 +1,77 @@
+The Numpy-Wavefunction device
+=============================
+
+The ``forest.numpy_wavefunction`` device provides an interface between PennyLane
+and the pyQuil `NumPy wavefunction simulator <http://docs.rigetti.com/en/stable/wavefunction_simulator.html>`_.
```

Comment:
The link seems to be broken here. This one seems to refer to the docs:
https://pyquil-docs.rigetti.com/en/v2.24.0/apidocs/autogen/pyquil.numpy_simulator.NumpyWavefunctionSimulator.html

However, it includes the version number and could otherwise be subject to a change of URL eventually breaking the link again. Maybe good to go without a link here?
```suggestion
The ``forest.numpy_wavefunction`` device provides an interface between PennyLane
and the pyQuil `NumpyWavefunctionSimulator`.
```